alpine-nix-rails-nginx/Dockerfile

```diff
@@ -0,0 +1,53 @@
+# dkubb/alpine-nix-rails-nginx
+
+FROM dkubb/alpine-nix:latest
+MAINTAINER Dan Kubb <dkubb@fastmail.com>
+
+EXPOSE 80
+
+# Configure nix
+COPY config.nix /root/.nixpkgs/config.nix
+
+# Install system dependencies
+RUN nix-env --install --fallback \
+    acl-2.2.52 \
+    bundler-1.10.5 \
+    gcc-wrapper-5.2.0 \
+    git-2.5.0 \
+    gnumake-4.1 \
+    libressl-2.2.2 \
+    libxml2-2.9.2 \
+    libxslt-1.1.28 \
+    nginx-1.8.0 \
+    nodejs-0.12.7 \
+    ruby-2.2.2-p0 \
```

Comment:
The nix ruby 2.2.2 release has most of the 2.2.3 patches applied to it, and specifically applies the segfault patch. I'd rather use the nix package than make our own and have to maintain it.